src/server/services/import/import.js

```diff
@@ -29,6 +28,16 @@ const importData = async (dataRaw, { slug, format, user, idField }) => {
   let data = await parseInputData(format, dataRaw, { slug });
   data = toArray(data);
 
+  const results = [];
+
+  for (const datum of data) {
+    results.push(await createCandidate(datum, { slug, user, idField }));
```

Comment:
Possible alternative option: start a db transaction so that either all of the imported users get created, or none of them, as leaving it in a state of "half of the users got imported and some failed" while giving an error response isn't a very ideal situation